### PR TITLE
Add support for custom issue resolved status

### LINKED ISSUE #60: Add support for custom issue resolved status
How about add support for custom issue resolved status ?

```
/jira.cfg
[DEFAULT]
url = https://jira.atlassian.com
username = USERNAME (or blank for no authentication
password = PASSWORD (or blank for no authentication)
# ssl_verification = True/False
# version = foo-1.0
# components = com1,second component,com3
# strategy = [open|strict|warn|ignore] (dealing with not found issues)
# docs_search = False (disable searching for issue id in docs)
# issue_regex = REGEX (replace default `[A-Z]+-[0-9]+` regular expression)
# resolve_status = ready-for-test, ...
```

### PR DESCRIPTION
**Pull request by lukas-bednar**
Fixes #60

### CODE REVIEW

**Comment by lukas-bednar** [comment]
Coverage decreased on module level, which is by design for pytest-cov plugin, and branches with DEFAULT_RESOLVED_STATUSES are not covered.

File: pytest_jira.py [hunk 1]
```diff
@@ -26,11 +27,16 @@ def __init__(
         marker,
         version=None,
         components=None,
+        resolved_statues=None,
```
Comment (Gordon-F) [inline pytest_jira.py]: Typo ?
s/resolved_statues/resolved_statuses/g

Comment (lukas-bednar) [inline pytest_jira.py]: yes, good eye, thanks

**Comment by lukas-bednar** [comment]
@KKoukiou done

**Pull request merged**